.github/workflows/push.yaml

```diff
@@ -91,6 +91,14 @@ jobs:
           go-version: ${{ env.GO_VERSION }}
           cache: false
 
+      - name: install operator-sdk
+        run: |
+          export ARCH=$(case $(uname -m) in x86_64) echo -n amd64 ;; aarch64) echo -n arm64 ;; *) echo -n $(uname -m) ;; esac)
+          export OS=$(uname | awk '{print tolower($0)}')
+          export OPERATOR_SDK_DL_URL=https://github.com/operator-framework/operator-sdk/releases/download/v1.34.1
+          curl -LO ${OPERATOR_SDK_DL_URL}/operator-sdk_${OS}_${ARCH}
+          chmod +x operator-sdk_${OS}_${ARCH} && sudo mv operator-sdk_${OS}_${ARCH} /usr/local/bin/operator-sdk
+
```

Comment:
I've noticed some active PRs are held up by this (see #75 for instance).  Maybe we can split this out so it can merge faster?  If you don't have the time, I can do it.